README.md

```diff
@@ -1,6 +1,201 @@
-# KubernetesAccessManager
-KubernetesAccessManager is a web application designed for multi-user management of Kubernetes cluster. It provides a graphical interface for users to manage their Kubernetes resources. Administrators can define precise access rights based on roles from an identity provider, such as Keycloak.
+<h1 style="display: flex; align-items: center; gap: 30px;">
+  Kubernetes Access Manager (KAM)
+  <img src="./docs/images/KAM_logo.png" alt="KAM Logo" style="height: 125px;">
+</h1>
 
-## Project Status
+---
+## For English Press [Here](#documentation-in-english) / Для русского нажмите [здесь](#документация-на-русском-языке)
 
-**Note:** This project is currently under development and is not ready for production use. We don't accept any contributions at this time.
+---
+## Dokumentacja w języku polskim
```

Comment:
Ja bym ten tytuł wywalił

Reply:
Ewentualnie coś innego, chwilowo pomysłu ni mam ale trochę kiepsko wygląda

Reply:
Też nie mam pomysłu, a jakiś tytuł musi być do przeklikiwania sie pomiędzy engielską i rosyjską dokumentacją, a jak jedynie w polskiej nie będzie to lekko łyso

Reply:
Tak jak jest jest git?